whirlpool/appliance.py

```diff
@@ -68,6 +69,11 @@ def _set_attribute(self, attribute, value, timestamp):
     def said(self):
         return self._said
 
+    def load_from_file (self):
+        with open("APISamples/Maytag - Dryer Sensing-Wet, Bulky Items cycle, Temp 4, Dryness 3, Wrinkle prevent low, extra power.json") as file:
```

Comment:
```suggestion
        with open("APISamples/mock_appliance_data.json") as file:
```
Lets make this a bit more generic. You could then just use a symbolic link named `mock_appliance_data.json` to choose which json to load.